README.md

```diff
@@ -1 +1,35 @@
-# protonmail-contacts
\ No newline at end of file
+# protonmail-contacts
+
+Proton Contacts built with React.
+
+Setting up the project:
+
+1. Create `appConfig.json` in the root of your project with:
```

Comment:
You should move the doc inside proton-pack and add a link here + custom key for the `client Id` (_which should be WebContacts I guess_)